src/Microsoft.VisualBasic.Core/tests/FinancialTests.cs

```diff
@@ -20,7 +21,7 @@ private static void AreEqual<T>(T expectedOld, T expectedNew, T actual)
             Assert.Equal(expected, actual);
         }
 
-        [Theory]
+        [ConditionalTheory(nameof(IsNotArmOrAlpine))] // some tests fail due to precision
```

Comment:
It would be better to allow some varience of the results, like it is done in other similar tests, e.g. https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System/Math.cs#L46

Reply:
I expect that we will find number of other configurations that do not return the exact same results as Windows Intel.

Reply:
Logged https://github.com/dotnet/corefx/issues/39393.